nautobot/core/templates/utilities/templatetags/utilization_graph.html

```diff
@@ -1,7 +1,7 @@
 <div {% if utilization_count and total_count %}title="Used: {{ utilization_count }}&#13;Count: {{ total_count }}" {% endif %}class="progress text-center">
-    {% if utilization < 30 %}<span style="font-size: 12px;">{{ utilization }}%</span>{% endif %}
-    <div class="progress-bar progress-bar-{% if utilization >= danger_threshold %}danger{% elif utilization >= warning_threshold %}warning{% else %}success{% endif %}"
+    {% if utilization < 30 %}<span>{{ utilization }}%</span>{% endif %}
+    <div class="progress-bar bg-{% if utilization >= danger_threshold %}danger{% elif utilization >= warning_threshold %}warning{% else %}success{% endif %}"
         role="progressbar" aria-valuenow="{{ utilization }}" aria-valuemin="0" aria-valuemax="100" style="width: {% if utilization > 100 %}100{% else %}{{ utilization }}{% endif %}%">
```

Comment:
Mildly curious why we have the `if utilization > 100` condition in there, seems like it'd be a bug if we ever actually encounter that case. Not blocking.

Reply:
Yup, sounds like #6151. Working on addressing it this sprint. :)